dashboard/src/app/business/block/volumeList.component.ts

```diff
@@ -323,9 +324,10 @@ export class VolumeListComponent implements OnInit {
                 "replicationPeriod":Number(this.repPeriod),
                 "secondaryVolumeId":res.json().id
             }
+            this.createReplicationDisplay = false;
+            this.getVolumes();
             this.ReplicationService.createReplication(param).subscribe((res) => {
                 this.getVolumes();
```

Comment:
You already called "this.getVolumes()" on line 328, is line 330 redundant?

Reply:
I think it should be the grammar of nodeJS